allauth/account/tests.py

```diff
@@ -301,27 +330,6 @@ def test_ajax_login_success(self):
         data = json.loads(resp.content.decode('utf8'))
         self.assertEqual(data['location'], '/accounts/profile/')
 
-    @override_settings(
-        ACCOUNT_EMAIL_VERIFICATION=app_settings.EmailVerificationMethod
-        .MANDATORY,
-        ACCOUNT_AUTHENTICATION_METHOD=app_settings.AuthenticationMethod
```

Comment:
Unless I'm missing something, `AccountTests` already defaults to `AuthenticationMethod.USERNAME`, and [the `@override_settings` decorator](https://docs.djangoproject.com/en/1.7/topics/testing/tools/#django.test.override_settings) overrides settings only for a method. So is it really necessary?